support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -79,11 +79,11 @@ object SendAcquisitionEvent {
           case Annual => thrift.PaymentFrequency.Annually
         }
 
-      def paymentProviderFromPaymentMethod(paymentMethod: PaymentMethod): thrift.PaymentProvider =
+      def paymentProviderFromPaymentMethod(paymentMethod: PaymentMethodDisplayFields): thrift.PaymentProvider =
         paymentMethod match {
-          case _: CreditCardReferenceTransaction => thrift.PaymentProvider.Stripe
-          case _: PayPalReferenceTransaction => thrift.PaymentProvider.Paypal
-          case _: DirectDebitPaymentMethod => thrift.PaymentProvider.Gocardless
+          case CreditCardDisplayFields => thrift.PaymentProvider.Stripe
+          case PaypalDisplayFields => thrift.PaymentProvider.Paypal
```

Comment:
This is a very simple change but I don't know how to test this. 
Is there a way to make sure this lambda still works as expected in CODE ?